tests/test_kpcovr.py

```diff
@@ -25,7 +25,9 @@ def __init__(self, *args, **kwargs):
         self.X = SFS().fit_transform(self.X)
         self.Y = SFS(column_wise=True).fit_transform(self.Y)
 
-        self.model = lambda mixing=0.5, **kwargs: KPCovR(mixing, alpha=1e-8, **kwargs)
+        self.model = lambda mixing=0.5, **kwargs: KPCovR(
+            mixing, alpha=1e-8, svd_solver=kwargs.pop("svd_solver", "full"), **kwargs
+        )
```

Comment:
Is this really enough to fix the problem? I'm just afraid that there are difficulties because of the way `self.Y` is created.